packages/terra-detail-view/src/DetailView.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import DetailViewSubtitle from './DetailViewSubtitle';
+import './DetailView.scss';
+
+const propTypes = {
+  /**
+   * The title display for the view
+   **/
+  title: PropTypes.element,
```

Comment:
Should the titled be opened up to allow a string? Currently this will throw a development warning when a string is passed in.